tests/test_wp.py

```diff
@@ -40,10 +42,20 @@ def test_mwp_basic_albert():
     answer = [WordPredictionResult(token_str='garlic', score=0.036625903099775314)]
     assert result == answer
 
+
 def test_mwp_basic_bert():
     happy_mwp = HappyWordPrediction("BERT", "bert-base-uncased")
     result = happy_mwp.predict_mask(
         "Please pass the salt and [MASK]",
     )
     answer = [WordPredictionResult(token_str='.', score=0.8466101884841919)]
-    assert result == answer
\ No newline at end of file
+    assert result == answer
+
+
+def test_mwp_basic_roberta():
+    happy_mwp = HappyWordPrediction("ROBERTA", "roberta-base")
+    result = happy_mwp.predict_mask(
+        "Please pass the salt and <mask>",  # Roberta requires mask to be '<mask>' as opposed to '[MASK]'
```

Comment:
If roberta, replace all [MASK] within the string to <mask>. That way we don't need separate documentation for ROBERTA

Reply:
Updated such that within the 'predict_mask' function the input text replaces instances of '[MASK]' with ''; such that the mask token is not changed for the input string yet the text is formatted in a way that RoBERTa likes is